### PR TITLE
esm_v2_eol_update

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/46267

## Description
updating support to match McAfee EOL https://kc.mcafee.com/corporate/index?page=content&id=KB94822

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/McAfee_ESM/Integrations/McAfee_ESM_v2/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
 Run queries and receive alarms from Intel Security ESM. Support version 10 and above.
```
Comment (Shellyber) [inline Packs/McAfee_ESM/Integrations/McAfee_ESM_v2/README.md:2]:
```suggestion
Run queries and receive alarms from Intel Security ESM.
```

**Review by Shellyber — CHANGES_REQUESTED** [review]
Nice.
We need to change also this part:
image

**Comment by bakatzir** [comment]
> Nice. We need to change also the `version` in this part: image

Are you certain? some customers might still use the EOL versions of ESM, and we stated in the readme the version we tested on. They can put even the EOL version, considering this is their responsibility, no? WDYT?

**Comment by Shellyber** [comment]
@bakatzir not sure we want to encourage/show the option to use the EOL versions. 
But if those versions still works..

**Review by Shellyber — APPROVED** [review]

**Pull request merged**